### PR TITLE
Remove-EntraScopedRoleMembership

### PR DESCRIPTION
**Pull request by v-uansari**

### CODE REVIEW

File: module/Entra/AdditionalFunctions/Remove-EntraScopedRoleMembership.ps1 [hunk 1]
```diff
@@ -0,0 +1,82 @@
+function Remove-EntraScopedRoleMembership {
```
[the remark this answers is not in the document]
Comment (v-pughosh) [reply module/Entra/AdditionalFunctions/Remove-EntraScopedRoleMembership.ps1:4]: done

File: test/module/Entra/Remove-EntraScopedRoleMembership.Tests.ps1 [hunk 2]
```diff
@@ -0,0 +1,38 @@
+BeforeAll {
```
[the remark this answers is not in the document]
Comment (v-pughosh) [reply test/module/Entra/Remove-EntraScopedRoleMembership.Tests.ps1:4]: done

**Review by emmanuel-karanja — APPROVED** [review]

File: module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/Remove-EntraScopedRoleMembership.md [hunk 3]
```diff
@@ -26,27 +24,37 @@ Removes a scoped role membership.
...
+}
+Remove-EntraScopedRoleMembership @params
 ```
 
-Removes scoped membership.
+This cmdlet removes a specific scoped role membership from Microsoft Entra ID.
+
+- `ObjectId` - specifies the object (ID) which you want to remove. In this example, `aaaaaaaa-2222-bbbb-aaaa-cccccccccccc` represents the ID of the Administrative Unit.
+
+- `-ScopedRoleMembershipId` - specifies the unique identifier (ID) of the scoped role membership, which you want to remove. In this example, `aaaaaaaa-bbbb-1111-aaaa-ddddddddddd` represent the ID of the ScopedRoleMembership.
```
Comment (SteveMutungi254) [inline module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/Remove-EntraScopedRoleMembership.md:53]:
```suggestion
- `ScopedRoleMembershipId` - This parameter specifies the unique identifier (ID) of the scoped role membership that you want to remove. In this example, aaaaaaaa-bbbb-1111-aaaa-ddddddddddd represents the ID of the ScopedRoleMembership. To obtain the details of a scoped role membership, you can use the `Get-EntraScopedRoleMembership` command
```

**Review by SteveMutungi254 — APPROVED** [review]
Approved.

**Pull request merged**